routes/userRouter.js

```diff
@@ -0,0 +1,42 @@
+const { Router }=require('express');
+const { UserModel } = require('../MongoDBModels/User');
+
+const userRouter = new Router()
+
+userRouter.post('/user', async (req, res) => {
```

Comment:
for what do you create this routes ?
you already have this logic in controllers/authController.js